src/main/charts/bitbucket/templates/job-mesh-add-nodes.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if and .Values.bitbucket.sysadminCredentials.secretName .Values.bitbucket.mesh.enabled .Values.bitbucket.mesh.nodeAutoRegistration -}}
+{{- range $index := until (.Values.bitbucket.mesh.replicaCount | int) }}
+{{- with $ }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-mesh-register-job-{{ $index }}
+  labels:
+    app.kubernetes.io/name: {{ include "common.names.name" . }}-mesh-job
+    {{- include "common.labels.commonLabels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-10"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      annotations:
+        {{- include "bitbucket.mesh.podAnnotations" . | nindent 8 }}
+      labels:
+        app.kubernetes.io/name: {{ include "common.names.name" . }}-mesh-job
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        {{- include "bitbucket.mesh.podLabels" . | nindent 8 }}
+    spec:
+      containers:
+        - name: bitbucket-mesh-register-services
+          image: curlimages/curl
+          command: ["/scripts/register-mesh-node.sh"]
```

Comment:
https://github.com/atlassian/data-center-helm-charts/pull/501/files#diff-68968aa1d23eb550c505be4c9bfac3f64becf9ac17e4e9ec0c7f204b32b087ec

Having commands in a configmap is better, especially if response code need to be checked.